src/index.html

```diff
@@ -7,8 +7,48 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="image__nav" href="#">
+        <img class="logo__img"
+        src="./images/logo.png"
+        alt="logo"
+        >
```

Comment:
```suggestion
        <img 
          class="logo__img"
          src="./images/logo.png"
          alt="logo"
        >
```
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md):
13. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.